samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/QuestionScoreListener.java

```diff
@@ -548,22 +548,39 @@ public boolean questionScores(String publishedId, QuestionScoresBean bean,
 				results.setItemGradingArrayList(answerList);
 				// The list is sorted by item id so that it will come back from the student in a 
 				// predictable order. This is also required by the getCalcQResult method.
-				Collections.sort(answerList, new Comparator<ItemGradingData>() {
-					public int compare(ItemGradingData i1, ItemGradingData i2) {
+				if (bean.getTypeId().equals("15")) { // CALCULATED_QUESTION
```

Comment:
This should import and use
import org.sakaiproject.tool.assessment.data.ifc.shared.TypeIfc;

TypeIfc.CALCULATED_QUESTION

Reply:
And it should be on the right side of the comparison as it's a constant!

Reply:
Done

Reply:
Hi @ern @mpellicer @juanjmerono @jonespm 

The requested changes are done by @maramar7. Could you please merge this PR? Thanks!